src/app/audiologist-login/audiologist-login.component.ts

```diff
@@ -14,11 +15,11 @@ import { ThsDataService } from '../services/ths-data.service';
 
 export class AudiologistLoginComponent {
   // Added audiologistUserName & changed (patientId --> audiologistID)
-  public audiologistUserName: string = ''; 
+  public audiologistUserName: string = '';
   public audiologistID: string = '';
   public authenticationFlag: boolean = true;
 
-  constructor(private router: Router, private tsDataService: TsScreenerDataService, private tfiDataService: TfiDataService, private thsDataService: ThsDataService) {};
+  constructor(private router: Router, private tsDataService: TsScreenerDataService, private tfiDataService: TfiDataService, private thsDataService: ThsDataService, private serverAuthenticationService: ServerAuthenticationService) { };
```

Comment:
If we are already logged in, then redirect to audiologist page.

Reply:
Adding a check here for a heartbeat creates a brief moment where the Login screen will load before redirect takes place.
@gstone271 : By the way, would I be misusing the heartbeat API here?

Reply:
Summary from team discussion: YES. Don't use API calls to make redirects.